tidy3d/web/api/autograd/autograd.py

```diff
@@ -886,6 +887,9 @@ def _run_tidy3d(
         verbose = run_kwargs.get("verbose", False)
         upload_sim_fields_keys(run_kwargs["sim_fields_keys"], task_id=job.task_id, verbose=verbose)
     path = run_kwargs.get("path", DEFAULT_DATA_PATH)
+    if task_name.endswith("_adjoint"):
```

Comment:
Another annoying thing is that this is not at all foolproof @tylerflex . I could very easily have given a task name that already ends in `"_adjoint"` if I'm doing some adjoint runs, before the extra `"_adjoint"` is appended for the bwd sim...

Reply:
I don't think there is a easy way to avoid this, how about adding an error message when starting simulations with this postfix?

Reply:
yea, it's unfortunate but we might be stuck with it. I like that solution.

Reply:
@txdai just squash your commits we can add the check later

Reply:
Ah sorry you did already